test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -21,86 +21,124 @@
+import java.awt.event.ActionListener;
 import java.beans.PropertyChangeEvent;
 import java.beans.PropertyChangeListener;
 import java.lang.reflect.InvocationTargetException;
 
+import javax.swing.JCheckBox;
 import javax.swing.JFileChooser;
 import javax.swing.JFrame;
-import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
 import javax.swing.SwingUtilities;
 import javax.swing.WindowConstants;
 
-/** @test
-  * @bug 8281966
-  * @key headful
-  * @library /java/awt/regtesthelpers
-  * @build PassFailJFrame
-  * @summary Test to check if the absolute path of Symbolic Link folder
-  *          is valid on ValueChanged property listener.
-  * @run main/manual FileChooserSymLinkTest
-*/
-
+/*
+ *@test
+ * @bug 8281966
+ * @key headful
+ * @requires (os.family == "windows")
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Test to check if the absolute path of Symbolic Link folder
+ *          is valid on ValueChanged property listener.
+ * @run main/manual FileChooserSymLinkTest
+ */
 public class FileChooserSymLinkTest {
     static JFrame frame = null;
+    static Boolean bMultiSel_Enabled = false;
+    static JFileChooser jfc = null;
+    static JPanel panel = null;
+    static JTextArea textArea = null;
+    static JCheckBox checkMSelection = null;
     static PassFailJFrame passFailJFrame = null;
 
     public static void main(String[] args) throws Exception {
-        Robot robot = new Robot();
         SwingUtilities.invokeAndWait(new Runnable() {
             public void run() {
                 try {
                     initialize();
-                } catch (InterruptedException e) {
-                    throw new RuntimeException(e);
-                } catch (InvocationTargetException e) {
+                } catch (InterruptedException | InvocationTargetException e) {
                     throw new RuntimeException(e);
                 }
             }
         });
-
-        robot.delay(2000);
-        robot.waitForIdle();
-
         passFailJFrame.awaitAndCheck();
     }
 
     static void initialize() throws InterruptedException, InvocationTargetException {
+        //Initialize the components
         final String INSTRUCTIONS = """
                 Instructions to Test:
-                1. Create a regular directory in any specific path.
-                    ex: mkdir C:\\target
-                2. Create a Symbolic link targeting the created test directory.
-                    ex : mklink /D C:\\link C:\\target
-                3. In JFileChooser, navigate to "link" created directed.
-                4. On click of the "link" directory, if the Absolute path of
-                    Symbolic Link is valid then Click PASS, else if it is
-                    null then Click FAIL.(Including MultiSelection Mode SET)
+                1. Open an elevated Command Prompt.
+                2. Paste the following commands:
+                cd /d C:\
```

Comment:
```suggestion
                cd /d C:\\
```

Another backslash is missing to display the backslash.